admin/create-amqp-extension

```diff
@@ -6,16 +6,20 @@ set -e -u
 source "$(dirname "${BASH_SOURCE[0]}")/lib/common.inc.bash"
 
 HELP=$(cat <<EOH
-Usage: $SCRIPT_NAME
+Usage: $SCRIPT_NAME [--help]
 
 Create 'amqp' extension (with broker) in the database.
 EOH
 )
 
-if [ $# -ne 0 ]
+if [[ $# -ne 0 && $1 =~ -*h(elp)? ]]
 then
-  echo >&2 "$SCRIPT_NAME: too many arguments"
-  echo >&2 "$HELP"
+  echo "$HELP"
+  exit
```

Comment:
```suggestion
  exit 0
```

Reply:
Changed in https://github.com/metabrainz/musicbrainz-docker/pull/131/commits/edb9b3b5fd0c25fdd0cf3eb017b6455d226da57c along with comments to document each exit status.